gradle.properties

```diff
@@ -1,3 +1,3 @@
-org.gradle.parallel=false
+org.gradle.parallel=true
```

Comment:
No strong opinion on this, but was recommended by the Gradle colleagues.

Reply:
IIRC this also runs tests in parallel, and if someone attempts to build the whole project locally, they would be running a lot of heavy tests in parallel - this needs verification tho

Reply:
When I discussed this with Gradle, they gave me the information, that this won't run the tests itself in one project in parallel, however, I agree that we maybe should again measure the effect on this on a full build (e.g. running all subprojects in parallel). 

The way I tested it (on our CI and running only `core` build) it had no negative effect. 

We can also rollback this individual change for now.

Reply:
yeah, it won't change how tests are running in a single module, but tests from multiple modules (core, kafka, db2, ...) will run in parallel IIRC, so I would recommend reverting it (unless we verify that it isn't the case, or find a way to prevent it

Reply:
Indeed those tests could be run in parallel. It is not a real issue with the way we currently fan out CI jobs, but it still seems potentially bad for users running the whole project. So let's roll back and only activate after conscious evluation.

Reply:
Just to add a data point, I've had dev machines that were unable to run our build in parallel without also limiting the number of workers. The docker VM can often be undersized compared to the number of host machine cores, and some of the containers we run are too greedy to run side-by-side when other modules are trying to run their tests too.

Reply:
Thanks for sharing, definitely better to keep this off then 👍 
The experiments we run did not really consider this, since the CI is set up in a way where this does not apply.

It is also interesting how more stable and reliable task caching will influence this.

Reply:
alternative: turn it on on CIs only

Reply:
Let's leave it off, have our CI generate more data, and then we can measure the impact (i.e. improvement, if any) better of turning it on on CI.